src/main/java/mate/academy/bookshop/dto/CreateBookRequestDto.java

```diff
@@ -1,15 +1,26 @@
 package mate.academy.bookshop.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
+import mate.academy.bookshop.validation.Isbn;
 
 @Data
 public class CreateBookRequestDto {
     private Long id;
+    @NotNull
     private String title;
```

Comment:
For text fields I would suggest to validate length(size) too. Because you have length restrictions on DB level, so it's better to check it here too.